modules/launcher/src/main/java/org/mule/module/launcher/ArtifactDeploymentListenerAdapter.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.launcher;
+
+
+import static org.mule.config.bootstrap.ArtifactType.APP;
+import static org.mule.config.bootstrap.ArtifactType.DOMAIN;
+import org.mule.api.MuleContext;
+import org.mule.config.bootstrap.ArtifactType;
+
+/**
+ * Adapts a {@link DeploymentListener} to an {@link ArtifactDeploymentListener}.
```

Comment:
Is not the opposite? (ie: adapts a ArtifactDeploymentListener to work as DeploymentListener)